src/main/java/com/tlswe/awsmock/ec2/control/MockEC2QueryHandler.java

```diff
@@ -501,8 +498,14 @@ public void handle(final Map<String, String[]> queryParams,
                             int minCount = Integer.parseInt(queryParams.get("MinCount")[0]);
                             int maxCount = Integer.parseInt(queryParams.get("MaxCount")[0]);
 
+                            final String[] subnetIdArray = queryParams.get("SubnetId");
+                            String subnetId = null;
+                            if (subnetIdArray != null) {
+                                subnetId = subnetIdArray[0];
```

Comment:
What if `SubnetId` is not provided? Can we allocate a random uuid as the subnetId for an instance?

Reply:
Another possibilty would be, that we use the MOCK_SUBNET_ID as default. 
What do you think?

Reply:
I agree that we use MOCK_SUBNET_ID as default.